CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ option(BUILD_LIBRETRO "libretro Enabled" ${BUILD_LIBRETRO_DEFAULT})
 option(BUILD_DEMO_CARTS "Demo Carts Enabled" ${BUILD_DEMO_CARTS_DEFAULT})
 option(BUILD_PRO "Build PRO version" FALSE)
 option(BUILD_PLAYER "Build standalone players" ${BUILD_PLAYER_DEFAULT})
+option(BUILD_COLLAB "Build with collaboration feature" OFF)
```

Comment:
Pls replace `OFF` in the CMakeLists.txt file with "``{BUILD_COLLAB_DEFAULT}".

Reply:
Oh hey, do you think you guys will merge this if I resolve conflicts and bring it up to date? It would be so cool to have for game jams and teaching.

Reply:
Hi,
Sure, we will merge it, also I think it would be great if you add a `--collab` command-line option to enable/disable the feature.
The list with the options is here https://github.com/nesbox/TIC-80/blob/master/src/studio/studio.h#L73.

Thank you.

Reply:
Ok, I'll find some time, thanks!

Reply:
Hey guys, just wanted to ask, any chance for a real-time mode? Without pulls/push/commits, all changes visible instantly.

Reply:
Hey, thanks for the suggestion - I'd have to play around with it to see how well it adapts to the replication architecture I setup, it might be fine or it might require some lockstep synchronization through the server to avoid trashing each others work if you put it under duress. It'd also probably want some visualization to see where the other user's "cursor" is. 

Have you tried a build with the current implementation? I find it works pretty well but I like to hear others opinions too.

Reply:
Very cool, @wadetb! <3 For a future refinement, make sure to look into algorithms like [Operational Transform](https://en.wikipedia.org/wiki/Operational_transformation) or [CRDTs](https://en.wikipedia.org/wiki/Collaborative_real-time_editor), which are made exactly for real-time editing cases like this! And yes, cursor awareness! Would absolutely *love* to use this with kids, I often teach or mentor young people who are just learning how to program!